Dockerfile

```diff
@@ -24,7 +24,7 @@
 # can't be run as these are just packages in this repo.
 
 # Build base has the packages installed that we need.
-FROM artifactory.algol60.net/docker.io/library/golang:1.16-alpine AS build-base
+FROM artifactory.algol60.net/docker.io/library/golang:1.23.3-alpine3.20 AS build-base
```

Comment:
Would specifying ```artifactory.algol60.net/docker.io/library/golang:1.23-alpine``` here allow for auto-updates of any 1.23.x sub-version vs pinning to 1.23.3?  When I updated to Go 1.23 in other repos I was going down the path of 1.23 vs pinning to a specific 1.23 version.

Reply:
Thanks, I changed it to artifactory.algol60.net/docker.io/library/golang:1.23-alpine